docs/pages/database-access/guides/azure-postgres-mysql.mdx

```diff
@@ -77,51 +68,84 @@ Create the Database Service configuration, specifying a region like this:
   ```code
   $ teleport db configure create \
     -o file \
-    --proxy=teleport.example.com:3080 \
+    --proxy=teleport.example.com:443 \
     --token=/tmp/token \
     --azure-mysql-discovery=eastus
   ```
 
 </TabItem>
-</Tabs>
 
-</ScopedBlock>
-<ScopedBlock scope={["cloud"]}>
-
-<Tabs>
-<TabItem label="PostgreSQL">
+<TabItem label="PostgreSQL and MySQL">
 
   ```code
   $ teleport db configure create \
     -o file \
-    --proxy=teleport.example.com:3080 \
+    --proxy=teleport.example.com:443 \
     --token=/tmp/token \
+    --azure-mysql-discovery=eastus \
     --azure-postgres-discovery=eastus
   ```
 
-</TabItem>
-<TabItem label="MySQL">
+<Admonition type="tip">
+This will create two `types` entities in `teleport.yaml`, one for each database
+type. This is useful if you want different regions, tags, or labels for each
+database type.
 
-  ```code
-  $ teleport db configure create \
-    -o file \
-    --proxy=teleport.example.com:3080 \
-    --token=/tmp/token \
-    --azure-mysql-discovery=eastus
-  ```
+Alternatively, you can edit `teleport.yaml` to include both database types in a
+single entry:
+
+```yaml
+db_service:
+  azure:
+  - types: ["mysql", "postgres"]
+  ...
+```
+
+</Admonition>
 
 </TabItem>
-</Tabs>
 
-</ScopedBlock>
+</Tabs>
 
-The command will generate a Database Service configuration with Azure MySQL/Postgres
+This command will generate a Database Service configuration with Azure MySQL/Postgres
 database auto-discovery enabled in the `eastus` region and place it at the
 `/etc/teleport.yaml` location.
 
-### Create a Teleport user
+### Create a Teleport role
+
+On your workstation logged in to your Teleport cluster with `tsh`, define a new
+role to provide access to your Azure database. Our example file is
+`azure-databse-role.yaml`:
+
+```yaml
+version: v5
+kind: role
+metadata:
+  name: azure-database-access
+spec:
+  allow:
+    db_labels:
+      '*': '*'
+    db_names:
+    - teleport-mysql
+    db_users:
+    - 'teleport'
```

Comment:
Why quote teleport here but not teleport-mysql above?

Reply:
No good reason. Adjusted, and `teleport-mysql` was removed anyway.